lib/core/mount.js

```diff
@@ -168,9 +169,13 @@ function mount(mountPath, parentApp, events) {
 
 		// Initialize the session store
 		try {
-			
-			var SessionStore = require(sessionStore)(express);
-			
+      if(sessionStore === 'connect-redis') {
+				expressSession = require('express-session');
```

Comment:
Ah, they're being built from the branch. Okay, I was editing 2.x to make it available in 2.x. I can also port my changes over to the master branch, is there an established process for doing this? Or, do I just port them there by hand?